### PR TITLE
Quickfix to linking from https://github.com/tessel/t2-docs/pull/162

### PR DESCRIPTION
**Pull request by Frijol**
I merged https://github.com/tessel/t2-docs/pull/162 without testing locally (lesson: always test locally!) and then found the image was showing up all

screenshot of broken image

in the published version of the docs. So I went in to fix it and found a few other problems.

This PR accomplishes a few fixes:

1. I tried out the Gitbook graphical editor, which I think messed up the code of conduct badge linking, in a previous too-easy-to-commit-to-master commit. I fix that in this commit and probably won't use the Gitbook graphical editor for a while.
2. Evidently, Github wants you to append `?raw=true` to image files that it hosts– otherwise it will show you a whole Github page of the image. The side effect of this is that without `?raw=true`, markdown won't parse the image as an image. This fixes the broken image screenshot shown above.
3. I realized in the course of the fix that the pindiagram.jpg image was in the root directory, but it belonged in the images folder– I moved it there.
4. Moving the file into the images folder meant re-editing the filepath. Instead of having it query github for the image, I realized it would be more efficient to give the file a relative link (since the file is stored in the same directory). This meant I didn't need the `?raw=true` bit anymore. This also made clear that the link was previously referencing the 276linesofcode branch's path to the image, rather than the t2-docs repo's image, which had the potential to become a very confusing bug if someone were to change the image in the future and see it fail to update (because it was linking to an image stored elsewhere)– so relative linking fixes three problems at once.

Also, gitbook and the github repo had become decoupled somehow, so I re-linked them; new updates seem to be working fine now.

### CODE REVIEW

**Comment by Frijol** [comment]
cc @mittalshravika @brihijoshi

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Tessel 2 Docs
 
-[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](https://github.com/tessel/project/blob/master/CONDUCT.md)
+![Code of Conduct](https://img.shields.io/badge/❤-code of conduct-blue.svg?style=flat)(https://github.com/tessel/project/blob/master/CONDUCT.md)
```
Comment (HipsterBrown) [inline README.md]: This doesn't appear to render correctly without the spaces encoded:

screen shot 2017-08-06 at 8 18 30 pm

Comment (Frijol) [inline README.md]: ah weird, it does locally– will fix

**Comment by Frijol** [comment]
@HipsterBrown fixed, good to merge?

**Comment by HipsterBrown** [comment]
Merge when ready @Frijol :+1:

**Pull request merged**